packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3631,15 +3631,19 @@ Factors have the following properties:
 The following factor types are supported:
 
 | Factor Type           | Description                                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------------------  |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `call`                | Software [OTP](http://en.wikipedia.org/wiki/One-time_password) sent via Voice Call to a registered phone number     |
+| `email`               | Software [OTP](http://en.wikipedia.org/wiki/One-time_password) sent via email                                       |
```

Comment:
Micro nit:  in "sent via email", is lowercase "email" what we want, or do we want "Email"?  (I don't really have an opinion, or not a useful one).

Reply:
I see no reason to capitalize "email".